server/data/manual-stub.test.ts

```diff
@@ -0,0 +1,39 @@
+import superagent from 'superagent'
+
+const wiremockUrl = 'http://localhost:9091'
+const wiremockAdminUrl = `${wiremockUrl}/__admin`
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
I'd just use that 'ts-ignore' comment at the top of the file, then no need to add individual comments?